### PR TITLE
discord time

### PR DESCRIPTION
**Pull request by jackharrhy**
pls review then merge k thx bye

### CODE REVIEW

File: content/contents.lr [hunk 1]
```diff
@@ -18,4 +18,4 @@ We promote interest in all areas of Computer Science, and host social events and
 
 - [Facebook group](https://www.facebook.com/groups/2202189009/)
 - [Mailing list](mailing_list)
-- [Slack](https://muncssociety.slack.com)
+- [Discord](https://discord.muncssociety.com)
```
Comment (nint8835) [inline content/contents.lr]: Domain name is muncompsci.ca

File: templates/layout.html [hunk 2]
```diff
@@ -67,7 +67,7 @@
                     </ul>
                 </li>
                 <li class="depth-1">
-                    <a href="https://muncssociety.slack.com/">Slack Community</a>
+                    <a href="https://discord.muncssociety.com/">Discord Guild</a>
```
Comment (nint8835) [inline templates/layout.html]: Domain name is muncompsci.ca

Comment (jackharrhy) [reply templates/layout.html]: 3ac1c545e6a20ff3574be86855797f3b6117c821

File: content/contents.lr (hunk 1 above)
Comment (jackharrhy) [reply content/contents.lr]: 3ac1c545e6a20ff3574be86855797f3b6117c821

Comment (nint8835) [reply content/contents.lr]: Nope, still wrong

File: templates/layout.html (hunk 2 above)
Comment (nint8835) [reply templates/layout.html]: Nope, still wrong

Comment (jackharrhy) [reply templates/layout.html]: i am good code

Comment (jackharrhy) [reply templates/layout.html]: 648374bc6717a229849c6911603ce3219fe038c3

File: content/contents.lr (hunk 1 above)
Comment (jackharrhy) [reply content/contents.lr]: 648374bc6717a229849c6911603ce3219fe038c3

**Pull request merged**